src/theme/components/button.ts

```diff
@@ -3,19 +3,82 @@ import { defineStyleConfig } from "@chakra-ui/react";
 export const buttonTheme = defineStyleConfig({
   baseStyle: {
     backgroundColor: "primary.600",
+    borderRadius: "base",
+    // token?
     color: "white",
     px: 6,
     py: 3,
   },
   sizes: {
-    sm: {},
-    md: {},
-    lg: {},
+    sm: {
+      height: 11,
+    },
+    md: {
+      height: 12,
+    },
+    lg: {
+      height: 14,
+    },
   },
   variants: {
-    primary: {},
-    secondary: {},
-    tertiary: {},
+    primary: {
+      // token?
+      boxShadow: "0 1.5 1.5 0 rgba(35, 78, 82, 0.08)",
+      _disabled: {
+        backgroundColor: "primary.500",
+        color: "white",
+        // token?
+        opacity: 0.64,
+      },
+      _hover: {
+        backgroundColor: "brand.50",
+        // token?
+        border: "1px solid",
+        borderColor: "brand.800",
+        // token?
+        boxShadow: "0 1 1.5 0 rgba(217, 107, 39, 0.18)",
+        color: "gray.700",
+        fontWeight: "500",
+      },
+      _active: {
```

Comment:
You're correct to map `_active` to "selected" but I think you may be missing a box shadow here and in the active styling for the secondary button?